src/views/AdditionalInformation/RelatedRequestsTable.tsx

```diff
@@ -0,0 +1,341 @@
+import { GetSystemIntake } from 'queries/types/GetSystemIntake';
+import { GetSystemIntakeRelatedRequestsVariables } from 'queries/types/GetSystemIntakeRelatedRequests';
+import { formatDateLocal } from 'utils/date';
+import formatContractNumbers from 'utils/formatContractNumbers';
+import globalFilterCellText from 'utils/globalFilterCellText';
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon,
+  sortColumnValues
+} from 'utils/tableSort';
+import { NotFoundPartial } from 'views/NotFound';
+
+import { LinkedRequestForTable } from './linkedRequestForTable';
+
+const RelatedRequestsTable = ({
+  requestID,
+  pageSize = 10
+}: {
+  requestID: string;
+  pageSize?: number;
+}) => {
+  const { t } = useTranslation('admin');
+  // make api call
+  const { loading, error, data } = useQuery<
+    GetSystemIntake,
+    GetSystemIntakeRelatedRequestsVariables
+  >(GetSystemIntakeRelatedRequests, {
+    variables: { systemIntakeID: requestID },
+    fetchPolicy: 'cache-and-network'
+  });
+
+  const tableData: LinkedRequestForTable[] = useMemo(() => {
+    if (error !== undefined) {
+      return [];
+    }
+
+    if (loading) {
+      return [];
+    }
+
+    if (data === undefined || data.systemIntake === null) {
+      return [];
+    }
+
+    const {
+      systemIntake: { relatedIntakes, relatedTRBRequests }
+    } = data;
+
+    const requests: LinkedRequestForTable[] = [];
+
+    // handle related intakes
+    relatedIntakes.forEach(relatedIntake => {
+      requests.push({
+        id: relatedIntake.id,
+        contractNumber: formatContractNumbers(relatedIntake.contractNumbers),
+        process: 'IT Governance',
+        projectTitle: relatedIntake.requestName || '',
+        status: relatedIntake.decisionState,
+        submissionDate: relatedIntake.submittedAt || ''
+      });
+    });
+
+    // handle trb requests
+    relatedTRBRequests.forEach(relatedTRBRequest => {
+      requests.push({
+        id: relatedTRBRequest.id,
+        contractNumber: formatContractNumbers(
+          relatedTRBRequest.contractNumbers
+        ),
+        process: 'TRB',
+        projectTitle: relatedTRBRequest.name || '',
+        status: relatedTRBRequest.status,
+        submissionDate: relatedTRBRequest.createdAt
+      });
+    });
+    return requests;
+  }, [data, error, loading]);
+
+  const columns: Column<LinkedRequestForTable>[] = useMemo<
+    Column<LinkedRequestForTable>[]
+  >(() => {
+    return [
+      {
+        Header: t<string>('tableColumns.projectTitle'),
+        accessor: 'projectTitle',
+        Cell: ({
+          row,
+          value
+        }: {
+          row: Row<LinkedRequestForTable>;
+          value: LinkedRequestForTable['projectTitle'];
+        }) => {
+          let link: string;
+          if (row.original.process === 'TRB') {
+            link = `/trb/task-list/${row.original.id}`;
+          } else {
+            link = `/governance-task-list/${row.original.id}`;
+          }
```

Comment:
The links in the table should bring the user to the admin view of the request, not the task list

Reply:
nice! thanks! updated - i am looking at the below comments though about the link, but this has been fixed. thanks Patrick!